go.mod

```diff
@@ -43,4 +43,5 @@ replace (
 	k8s.io/client-go => k8s.io/client-go v11.0.0+incompatible
 	k8s.io/code-generator => k8s.io/code-generator v0.0.0-20190311093542-50b561225d70
 	k8s.io/component-base => k8s.io/component-base v0.0.0-20190708175518-244289f83105
+	k8s.io/helm => k8s.io/helm v2.14.3+incompatible
```

Comment:
Can you added a comment on why this overwrite is needed?

Reply:
Added